docs/migration.md

```diff
@@ -506,6 +506,9 @@ print(Model(x='1'))
 In Pydantic V1, the printed result would have been `x=1`, since the value would pass validation as an `int`.
 In Pydantic V2, we recognize that the value is an instance of one of the cases and short-circuit the standard union validation.
 
+To revert to the non-short-circuiting left-to-right behavior of V1, annotate the union with `Field(union_mode='left_to_right')`.
+See [Union Mode](./usage/types/unions.md#union-mode) for more details.
```

Comment:
I'm not an expert on V1's union validation, this statement may not be true. I thought it might be worth calling out this or something similar here in the migration guide.

Reply:
It's right